src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="shortcut icon"
+      href="images/logo.png"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="logo"
+        />
+      </a>
```

Comment:
why do you use the same class selector for both tags
```suggestion
      <a href="#">
        <img
          src="images/logo.png"
          alt="Logo"
          class="logo"
        />
      </a>
```